LPS-72193 In case of local staging there is no need to add exactly the same friendly URL entry

pavel-savinov opened the pull request.

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2010/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2010/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Some tests FAILED.</h1><p>Build Time: 2 hours 8 minutes 23 seconds 481 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7aef70d42cade6868677f6a5f71d099cd458041e">7aef70d42cade6868677f6a5f71d099cd458041e</a></p>119 out of 122 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2010/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141850/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</strong></strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152618/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle121-jdk8</strong></strike></a></li></ul></div><details><summary><strong>119 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174104/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/168476/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119159/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164597/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tcserver32-mysql56-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149205/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-db2105-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172168/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164598/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23852/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/165827/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172169/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-sybase160-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119126/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174105/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-websphere85-mysql56-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143938/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157793/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164599/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/168477/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172170/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/10</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143939/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/11</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157794/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/12</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/106629/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/13</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23854/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/14</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145212/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/15</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149207/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/16</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/170398/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/17</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141844/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/18</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143940/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/19</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119127/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/167513/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/20</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172171/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/21</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157796/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/22</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172172/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/23</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23855/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/24</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152611/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/25</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157795/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/26</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/106630/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/27</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145213/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/28</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149208/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/29</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/153021/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141845/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/30</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/31</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/100138/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/33</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/170399/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/34</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/167514/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/35</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23856/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/36</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145214/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/37</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149209/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/38</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152612/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/39</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157797/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/40</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/106631/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/41</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172174/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/42</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/165828/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/5</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155441/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/6</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174106/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149206/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23853/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/9</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145217/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172177/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152615/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119128/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/3</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172178/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/4</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164600/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157800/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174108/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/2</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/159482/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/3</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155444/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/4</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172188/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/153022/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152616/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/167518/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/168478/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119129/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172180/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152617/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143945/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141849/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/4</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/165829/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174109/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145219/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/2</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/159483/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157801/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/4</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164601/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155445/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/167520/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/153023/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119130/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/4</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/168479/">test-portal-acceptance-pullrequest-batch(master)/integration-db2105-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/159484/">test-portal-acceptance-pullrequest-batch(master)/integration-mariadb100-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145220/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23802/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle121-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/172181/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/143946/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase160-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174111/">test-portal-acceptance-pullrequest-batch(master)/javadoc-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/156636/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152619/">test-portal-acceptance-pullrequest-batch(master)/lpkg-override-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/165831/">test-portal-acceptance-pullrequest-batch(master)/lpkg-persistence-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155446/">test-portal-acceptance-pullrequest-batch(master)/lpkg-release-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149212/">test-portal-acceptance-pullrequest-batch(master)/lpkg-startup-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119131/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157802/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/167521/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/153024/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mariadb100-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164602/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/102353/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23859/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/165830/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174110/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/156639/">test-portal-acceptance-pullrequest-batch(master)/portal-frontend-js-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119133/">test-portal-acceptance-pullrequest-batch(master)/portal-license-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155448/">test-portal-acceptance-pullrequest-batch(master)/portal-startup-space-in-path-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149213/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/156638/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/170401/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155447/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/119132/">test-portal-acceptance-pullrequest-batch(master)/slim-profile-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/149214/">test-portal-acceptance-pullrequest-batch(master)/source-format-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145221/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141851/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/156637/">test-portal-acceptance-pullrequest-batch(master)/wsdd-builder-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/235/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-6.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2010/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2010/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>119 Jobs Passed.<br/>3 Jobs Failed.</p></div><pre><code>Downstream jobs FAILED.</code></pre></div></li><li><div><h5><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/141850/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></h5><div><h6>Job Results:</h6><p>4217 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/141850/">AXIS_VARIABLE=0,label_exp=!master #141850</a><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/141850//testReport/com.liferay.portal.language/LanguageImplTest$GetAvailableLocales/testCompanyThreadLocalIsDefaultWithNoArgs">LanguageImplTest$GetAvailableLocales.testCompanyThreadLocalIsDefaultWithNoArgs</a><pre><code>com.liferay.portal.kernel.exception.NoSuchResourceActionException: 1_WAR_tasksportlet#VIEW
	at com.liferay.portal.service.impl.ResourceActionLocalServiceImpl.getResourceAction(ResourceActionLocalServiceImpl.java:388)
	at com.liferay.portal.spring.aop.ServiceBeanMethodInvocation.proceed(ServiceBeanMethodInvocation.java:163)
	at com.liferay.portal.spring.aop.ServiceBeanAopProxy.invoke(ServiceBeanAopProxy.java:169)
	at com.sun.proxy.$Proxy86.getResourceAction(Unknown Source)
	at com.liferay.portal.service.impl.ResourcePermissionLocalServiceImpl.doUpdateResourcePermission(ResourcePermissionLocalServiceImpl.java:1283)
	at com.liferay.portal.service.impl.ResourcePermissionLocalServiceImpl.updateResourcePermission(ResourcePermissionLocalServiceImpl.java:1355)
	at com.liferay.portal.service.impl.ResourcePermissionLocalServiceImpl.setResourcePermissions(ResourcePermissionLocalServiceImpl.java:1158)
	at com.liferay.portal.spring.aop.ServiceBeanMethodInvocation.proceed(ServiceBeanMethodInvocation.java:163)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor.execute(DefaultTransactionExecutor.java:54)
	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:58)
	at com.liferay.portal.spring.aop.ServiceBeanMethodInvocation.proceed(ServiceBeanMethodInvocation.java:137)
	at com.liferay.portal.spring.aop.RetryAdvice.invoke(RetryAdvice.java:81)
	at com.liferay.portal.spring.aop.ServiceBeanMethodInvocation.proceed(ServiceBeanMethod...</code></pre></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152618/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle121-jdk8</a></h5><div><h6>Job Results:</h6><p>14 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/152618/">AXIS_VARIABLE=2,label_exp=!master #152618</a><pre><code>     [exec]     SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
     [exec]     SLF4J: Actual binding is of type [com.liferay.util.sl4fj.LiferayLoggerFactory]
     [exec] 
     [exec] com.liferay.dynamic.data.lists.service.persistence.test.DDLRecordSetPersistenceTest STANDARD_OUT
     [exec]     Loading jar:file:/opt/dev/projects/github/liferay-portal/bundles/tomcat-8.0.32/webapps/ROOT/WEB-INF/lib/portal-impl.jar!/system.properties
     [exec]     Loading jar:file:/opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.impl/2.0.0/f4446352654136205deaa0e4b1a854fd9eaeca78/com.liferay.portal.impl-2.0.0.jar!/system.properties
     [exec]     Loading file:/opt/dev/projects/github/liferay-portal/bundles/tomcat-8.0.32/webapps/ROOT/WEB-INF/classes/system-ext.properties
     [exec] 15:06:34,128 INFO  [elasticsearch[Fred Myers][scheduler][T#1]][jvm:178] [Fred Myers] [gc][young][1085][11865] duration [1.7s], collections [2]/[3.6s], total [1.7s]/[2.2m], memory [882.9mb]-&gt;[903.4mb]/[1023.9mb], all_pools {[young] [25.4mb]-&gt;[31.8mb]/[31.8mb]}{[survivor] [0b]-&gt;[0b]/[64kb]}{[old] [871mb]-&gt;[871.5mb]/[992mb]}
     [exec] 15:06:51,646 INFO  [elasticsearch[Fred Myers][scheduler][T#1]][jvm:178] [Fred Myers] [gc][young][1091][11890] duration [2.2s], collections [3]/[6.4s], total [2.2s]/[2.3m], memory [879.5mb]-&gt;[878.5mb]/[1023.9mb], all_pools {[young] [20.6mb]-&gt;[31.8mb]/[31.8mb]}{[survivor] [0b]-&gt;[0b]/[64kb]}{[old] [877.9mb]-&gt;[878.5mb]/[992mb]}
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:240: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1497: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-common.xml:634: /opt/dev/projects/github/liferay-portal/build-common.xml:636: exec returned: 137
      [get] Getting: http://test-1-9/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/152618//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20170926150659195.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20170926150659195.txt</code></pre></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></html>

pavel-savinov commented:
@Preston-Crary These test fails seems to be unrelated with the PR.
I fixed all the issues you mentioned in the email(each one in the separate commit) and also added an API for update of FriendlyURLEntry, to make it clearer.
Also, as I mentioned in Skype, there is no way to add multiple FriendlyURLs with the same combination of classNameId, classPK and urlTitle, in this case service throws DuplicateException and this behaviour is the same in many other places of Portal(for example, see DDMTemplateLocalServiceImpl, addTemplate method).

If you need any further clarification please ping me here or on email.

Thanks!

/cc @ealonso @shuyangzhou

Preston-Crary commented:
Please see https://github.com/shuyangzhou/liferay-portal/pull/4906
